docs/e2e.md

```diff
@@ -0,0 +1,152 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# End to End
```

Comment:
End to End testing

We could also rename the document to e2e-testing.md, but shorter names are nicer.